### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by karthikgeek1**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import "antd/dist/antd.css";
```
Comment (Souvikdeb2612) [inline src/App.js:1]: first all external imports then 1 space, then internal functions and component import, then 1 space and finally css imports should occur in the end

File: src/Components/Cards.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Card, Collapse } from 'antd';
+import './Cards.css'
```
Comment (Souvikdeb2612) [inline src/Components/Cards.jsx]: same as above

File: src/Components/Layout/StockLayout.jsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Layout } from "antd";
-
 import "../Layout/StockLayout.css";
```
Comment (Souvikdeb2612) [inline src/Components/Layout/StockLayout.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (karthikgeek1) [reply src/App.js:1]: sir isn't this an external import?

Comment (karthikgeek1) [reply src/App.js:1]: sir what are external imports?

Comment (Souvikdeb2612) [reply src/App.js:1]: anything that is imported from installed libraries and not created by you

Comment (karthikgeek1) [reply src/App.js:1]: sir so, import "antd/dist/antd.css"; is external right?

Comment (Souvikdeb2612) [reply src/App.js:1]: but it's a CSS file so it'll come in the end along with normal internal css

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**